conf/app.routes

```diff
@@ -2,7 +2,7 @@
 
 ->         /hmrc-frontend                               hmrcfrontend.Routes
 
-GET        /                                            controllers.IndexController.onPageLoad
+GET        /                                            controllers.Default.redirect(to = "/advance-valuation-ruling/applications-and-rulings")
```

Comment:
Would love to see a test for this. Would also be great if we could get rid of the absolute path.